app/code/Magento/Checkout/Model/Cart/ConfigProvider.php

```diff
@@ -0,0 +1,59 @@
+<?php
```

Comment:
Please specify strict return types for methods, remove the "@api" and "@codeCoverageIgnore" annotation and fix the phpdocs to be consistent (for serializer)